core/arch/arch.c

```diff
@@ -681,6 +681,7 @@ d_r_arch_init(void)
      * masks. Also priv_mcontext_t.opmask slots are AVX512BW wide.
      */
     IF_X86(ASSERT(sizeof(dr_opmask_t) == OPMASK_AVX512BW_REG_SIZE));
+    ASSERT(NUM_GP_REGS == DR_NUM_GPR_REGS);
```

Comment:
Exporting opnd.h to everybody would be a bigger change which should be done separately.  There may be circular dependencies blocking it: not sure.  We already work around it in other ways so if it can be made to work it might simplify some other things.  I filed #3794 and may try to do it separately soon.

Reply:
Ok.